packages/lib-content/src/screens/Home/DefaultHome/components/Introduction.js

```diff
@@ -0,0 +1,122 @@
+import { Box, Button, Paragraph } from 'grommet'
+import { useState } from 'react'
+import styled from 'styled-components'
+import { AuthModal, SpacedHeading } from '@zooniverse/react-components'
+import { useTranslation } from '../../../../translations/i18n.js'
+
+import Stats from '../../../../components/Stats/Stats.js'
+import SubHeading from '../../../../components/HeadingForAboutNav/SubHeading.js'
+
+const StyledSignIn = styled(Button)`
+  width: 47%;
+  border-radius: 5px;
+  border: solid 1px ${props => props.theme.global.colors['neutral-1']};
+  font-size: 0.8rem;
+  padding: 8px 5px;
+  text-align: center;
+  color: black;
+  background-color: white;
+  font-weight: normal;
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const StyledRegister = styled(Button)`
+  width: 47%;
+  border-radius: 5px;
+  border: solid 1px ${props => props.theme.global.colors['neutral-1']};
+  font-size: 0.8rem;
+  padding: 8px 5px;
+  text-align: center;
+  font-weight: normal;
+  color: white;
+  background-color: ${props => props.theme.global.colors['neutral-1']};
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const VideoWrapper = styled(Box)`
+  border-radius: 8px; // same as Stat component
+  aspect-ratio: 16 / 9;
+  overflow: hidden;
+  margin: 60px 0;
+`
+
+export default function Introduction() {
+  const { t } = useTranslation()
+  const [activeIndex, setActiveIndex] = useState(-1)
+
+  function openRegisterModal() {
+    setActiveIndex(1)
+  }
+
+  function openSignInModal() {
+    setActiveIndex(0)
+  }
+
+  function closeAuthModal() {
+    setActiveIndex(-1)
+  }
+
+  return (
+    <>
+      <AuthModal
+        activeIndex={activeIndex}
+        closeModal={closeAuthModal}
+        onActive={setActiveIndex}
+      />
+      <SpacedHeading
+        color={{ light: 'neutral-1', dark: 'accent-1' }}
+        level={2}
+        size='1.5rem'
+        alignSelf='center'
+        margin={{ top: '30px', bottom: '15px' }}
+      >
+        {t('Home.DefaultHome.headings.one')}
+      </SpacedHeading>
+      <SubHeading>{t('Home.DefaultHome.subheadings.one')}</SubHeading>
+      <Paragraph
+        color={{ light: 'black', dark: 'white' }}
+        margin={{ vertical: 'medium' }}
+      >
+        {t('Home.DefaultHome.Introduction.description')}
+      </Paragraph>
+      <Box direction='row' justify='between' margin={{ bottom: '60px' }}>
+        <StyledSignIn
+          label={t('Home.DefaultHome.Introduction.signIn')}
```

Comment:
I think "sign in" and "register" are 14px in Figma.

Reply:
Oops I just realized I skipped over these changes, I'll include them in https://github.com/zooniverse/front-end-monorepo/pull/6108!

Reply:
Following our default Storybook styling here, feel free to do so for all buttons!